### DISCUSSION TITLE
Anybody try to install dialoqbase in a folder?

### DISCUSSION THREAD

**Discussion by phpia** [n4ze3m/dialoqbase#221] [Q&A]
Hi Im trying to install dialoqbase throw nginx in a folder a serve throw proxy example.
mydomain.com/dialoqbase

Any experience?
Tahnks

**Comment by n4ze3m** [comment]
Hey, I tried serving the entire application via a single domain, which is easy—all you need is Docker

I haven't tested any other methods yet. Sorry about it.

What error are you getting?

**Comment by phpia** [comment]
Thanks Nazeem if I found an elegant way of deploy dialoqbase with nginx in a subforlder I will post here
